packages/website/src/lib/constants.ts

```diff
@@ -32,14 +32,9 @@ export function isValidVersionSlug (maybeSlug: string | undefined): maybeSlug is
 }
 
 export function algoliaEnv () {
-  const ALGOLIA_APP_ID = process.env.NEXT_PUBLIC_ALGOLIA_APP_ID
-  const ALGOLIA_SEARCH_API_KEY = process.env.NEXT_PUBLIC_ALGOLIA_SEARCH_API_KEY
-  const ALGOLIA_INDEX_NAME = process.env.NEXT_PUBLIC_ALGOLIA_INDEX_NAME
-
-  if (ALGOLIA_APP_ID === undefined || ALGOLIA_SEARCH_API_KEY === undefined || ALGOLIA_INDEX_NAME === undefined) {
-    console.log('algoliaEnv', ALGOLIA_APP_ID, ALGOLIA_SEARCH_API_KEY, ALGOLIA_INDEX_NAME)
-    throw new Error('Missing Algolia environment variables')
-  }
+  const ALGOLIA_APP_ID = process.env.NEXT_PUBLIC_ALGOLIA_APP_ID || 'VJ9GF38NJX'
```

Comment:
We should not hardcode these like this. It will make identifying errors more difficult in the future.